### PR TITLE
Receipt class

### PR DESCRIPTION
**Pull request by Jminding**

### CODE REVIEW

File: assignment.swift [hunk 1]
```diff
@@ -51,3 +51,13 @@
  For the final step, I would like you to submit a pull request.  Remember, a pull request is telling me that your code is ready for review and that you believe your code is ready for deployment (i.e. the final version).  Only at this point will I review your code and give you feedback.
  
  */
+
+class Receipt {
```
Comment (gavinmccabe) [inline assignment.swift:55]: Should be a `struct`

File: assignment.swift [hunk 2]
```diff
@@ -51,3 +51,13 @@
...
+    
+    var ID: Int
+    var prdouctName: String
+    var productPrice: Double
+    var isPaid: Bool
+
```
Comment (gavinmccabe) [inline assignment.swift:61]: Remove empty line

**Review by gavinmccabe — CHANGES_REQUESTED** [review]

**Comment by Jminding** [comment]
HOW ABOUT NO HAHAHAHA

**Review by gavinmccabe — APPROVED** [review]
Good job.  Merged.

**Pull request merged**